Helper/Account.php

```diff
@@ -107,7 +104,7 @@ public function __construct(
         $this->logger = $context->getLogger();
         $this->nostoHelperScope = $nostoHelperScope;
         $this->nostoHelperUrl = $nostoHelperUrl;
-        $this->urlBuilder = $urlBuilder;
+        $this->urlBuilder = $this->_urlBuilder;
```

Comment:
Should be `$context->getUrlBuilder()`. We want to specifically use the URL builder in context